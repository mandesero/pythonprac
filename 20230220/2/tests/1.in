addmon 1 1 Hello, it's a cow!
up
down
right
up

