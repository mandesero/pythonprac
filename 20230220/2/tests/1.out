Added monster to (1, 1) saying Hello, it's a cow!
Moved to (0, 1)
Moved to (0, 0)
Moved to (1, 0)
Moved to (1, 1)
 ____________________ 
< Hello, it's a cow! >
 -------------------- 
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
